### PR TITLE
Lost to follow up Report

### PR DESCRIPTION
**Pull request by solemabrothers**
@ssmusoke  please find the Lost to Follow up report here according the card assigned https://trello.com/c/BlOu9b9r/741-lostto-followup. Lost is coming your way shortly, am waiting for your review

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
+package org.openmrs.module.ugandaemrreports.reports;
+
+
+        import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java]: @solemabrothers These imports should not be indented

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        import org.openmrs.module.ugandaemrreports.library.HIVPatientDataLibrary;
+        import org.openmrs.module.ugandaemrreports.metadata.HIVMetadata;
+        import org.springframework.beans.factory.annotation.Autowired;
+        import org.springframework.stereotype.Component;
+
+        import java.util.ArrayList;
+        import java.util.List;
+        import java.util.Properties;
+
+/**
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java]: @solemabrothers Wrong class comment

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+ * Daily Appointments List report
+ */
+        @Component
+        public class SetUpLostToFollowUp extends UgandaEMRDataExportManager {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java]: @solemabrothers Rename to SetupEWILostToFollowupReport

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java (hunk 1 above)
Comment (solemabrothers) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostToFollowUp.java]: @ssmusoke  this has been fixed

**Pull request merged**